paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -1266,6 +1267,7 @@ def get_mesos_utilization_error(
     When the boost feature is enabled, the current_load will be artifically increased
     and stored into boosted_load. If the boost is disabled, boosted_load = current_load
     """
+    system_config = load_system_paasta_config()
```

Comment:
Can you pass system_paastsa_config from autoscale_local_cluster() instead of reading and parsing the same files?

Reply:
Good suggestion.